shards/egui/src/util.rs

```diff
@@ -63,7 +64,7 @@ where
   result
 }
 
-pub fn get_object_from_var_opt<'a, T>(
+pub unsafe fn get_object_from_var_opt<'a, T>(
```

Comment:
why unsafe?

Reply:
Because the type you're casting into could not be the type that is there.

Reply:
same for comments below